torchcfm/optimal_transport.py

```diff
@@ -55,12 +70,43 @@ def get_map(self, x0, x1):
         return p
 
     def sample_map(self, pi, batch_size):
```

Comment:
building on the above, this would become:
```
from typing import Tuple
def sample_map(self, pi: np.Array, batch_size: int) -> Tuple[np.Array, np.Array]:
    blah blah blah
```

or something very similar to that.